compiler/rustc_codegen_ssa/src/back/link.rs

```diff
@@ -3116,12 +3116,27 @@ fn add_lld_args(
 
     let self_contained_linker = self_contained_cli || self_contained_target;
     if self_contained_linker && !sess.opts.cg.link_self_contained.is_linker_disabled() {
+        // We tell `cc` where to find the lld wrappers, in a similar way to how we locate
+        // codegen-backends dylibs, in the tools search paths, which contain:
+        // - the sysroot's `rustlib` bin path,
+        // - and as a fallback for cases using `--sysroot` without tools: in the default host
+        //   sysroot where rustc is currently located.
```

Comment:
These comments are outdated now (or at least misplaced).

Reply:
They're just a summary of what happens, they're also in `get_tools_search_paths`. I guess I can remove them.

Reply:
Done